mmv1/api/resource.go

```diff
@@ -1830,3 +1833,36 @@ func (r Resource) ShouldGenerateSweepers() bool {
 	}
 	return true
 }
+
+func (r Resource) GithubURL() string {
+	if r.githubUrl != "" {
+		return r.githubUrl
+	}
+
+	r.githubUrl = GITHUB_BASE_URL + r.SourceYamlFile
+	return r.githubUrl
+}
+
+func (r Resource) CodeHeader(templatePath string) string {
+	templateUrl := GITHUB_BASE_URL + templatePath
+
+	return fmt.Sprintf(`// ----------------------------------------------------------------------------
```

Comment:
My initial thoughts are that this should be a subtemplate, but code/markdown difference is annoying enough that I'm okay with this